src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,10 @@
 
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T> {
```

Comment:
```suggestion
public interface MachineProducer {
List<? extends Machine> get();
```
[https://mate-academy.github.io/jv-program-common-mistakes/java-core/pecs/pecs.html](url)